cmake/Modules/Findlibusb-1.0.cmake

```diff
@@ -56,7 +56,7 @@ else()
             PATH_SUFFIXES libusb-1.0)
 
   find_library(LIBUSB_1_LIBRARY
-               NAMES usb-1.0
+               NAMES usb-1.0 libusb-1.0
```

Comment:
Please add a comment summarizing the commit

Reply:
To the commit itself? I thought I did that right but I guess it only added text to the PR? I guess that it isn't fixable with the web interface, I'll pull and amend.

Reply:
A line of comment saying: `lib prefix needed for Conan and on Windows` can be added to the file via web interface.

Reply:
Oh, so what you want is a CMake comment, not a commit message?

Reply:
Done.